Sqrt/Implement Sqrt(x) without math library.txt

```diff
@@ -0,0 +1,53 @@
+/*Implement int sqrt(int x).
+
+Compute and return the square root of x, where x is guaranteed to be a non-negative integer.
+
+Since the return type is an integer, the decimal digits are truncated and only the integer part of the result is returned.*/
+
+class Solution {
+public:
+    int mySqrt(long int x) {
+        
+        long int c=0,m;
+        for(long int i=1;i<=x;i++)
+        {
+            m=i*i;
+            if(m==x)
+            {
+               c=i;
+                break;
+            }
+            else if(m>x){
+                
```

Comment:
Remove the extra lines. You have not used new-line in `if` part of the block. Then maintain consistency in `else if` part also.